### PR TITLE
fix: add rakesh9100

### LINKED ISSUE #1665: [PROFILE ADDITION] <Rakesh Roshan>
### Profile screenshot

Screenshot 2023-01-18 000820

### PR DESCRIPTION
**Pull request by Rakesh9100**
resolves #1665 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

### CODE REVIEW

File: profiles/rakesh9100.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/73993775/212983418-aaad5362-7792-4125-8faa-c0a965eb3572.png",
+    "issueId": 1665,
+    "name": "Rakesh Roshan",
+    "username": "rakesh9100"
+  }
```
Comment (Panquesito7) [inline profiles/rakesh9100.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

File: profiles/rakesh9100.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/73993775/212983418-aaad5362-7792-4125-8faa-c0a965eb3572.png",
+    "issueId": 1665,
+    "name": "Rakesh Roshan",
+    "username": "rakesh9100"
```
Comment (Panquesito7) [inline profiles/rakesh9100.json:5]: Wrong indentation.
```suggestion
  "image": "https://user-images.githubusercontent.com/73993775/212983418-aaad5362-7792-4125-8faa-c0a965eb3572.png",
  "issueId": 1665,
  "name": "Rakesh Roshan",
  "username": "rakesh9100"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/rakesh9100.json (hunk 1 above)
Comment (Rakesh9100) [reply profiles/rakesh9100.json]: Done 😌

File: profiles/rakesh9100.json (hunk 2 above)
Comment (Rakesh9100) [reply profiles/rakesh9100.json]: Resolved 😉 @Panquesito7

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by Rakesh9100** [comment]
> Thanks! 🎉

Kindly merge @Panquesito7 Thanks!!

**Comment by Panquesito7** [comment]
> > Thanks! 🎉
> 
> Kindly merge @Panquesito7 Thanks!!

Will do so when another maintainer approves the PR.

**Comment by Rakesh9100** [comment]
Okay, and sorry for this requested review to you, sent by mistake @Panquesito7

**Comment by Rakesh9100** [comment]
Hello, @Panquesito7 pls request the second reviewer who he/she is to approve and merge so that my profile get visible on the website...

**Review by schmelto — APPROVED** [review]

**Pull request merged**